pkg/views/util/spinner.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/charmbracelet/bubbles/spinner"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	log "github.com/sirupsen/logrus"
+)
+
+type model struct {
+	spinner  spinner.Model
+	quitting bool
+}
+
+type Msg string
+
+func initialModel() model {
+	s := spinner.New()
+	s.Spinner = spinner.Dot
+	s.Style = lipgloss.NewStyle().Foreground(lipgloss.Color("205"))
```

Comment:
Can you change this to use the "Blue" color from the view styles module that is used for project names and similar secondary labels (instead of the default Charm fuchsia)

Reply:
color changed to 'Blue' as requested.